### DISCUSSION TITLE
https://mtex-toolbox.github.io/images/ODFImport_02.png

### DISCUSSION THREAD

**Discussion by JinkuiMeng** [mtex-toolbox/mtex#662] [Ask Anything]
According to the guidance of this webpage(https://mtex-toolbox.github.io/ODFImport.html), I can get the diagram of ODF, but Phi 1 changes from 0 to 360 degrees. **How can I limit the change range to 0 to 90 degrees?**

**Comment by ralfHielscher** [comment]
Restricting the Phi1 range makes only sense if you have specimen symmetry. If this is the case specify it, e.g. by
```matlab
ss = specimensSymmetry('orthorhombic')
odf = ODF.load('fname',cs,ss)
```
Now the range is automatically restricted.

Ralf

**Reply by JinkuiMeng** [reply]
Dear Ralf,
My material is alpha Ti, and the codes I edit are:
cs = crystalSymmetry('hexagonal');
odf = ODF.load (fname,cs,'resolution',res,'Bunge',...
'ColumnNames',{'Euler 1','Euler 2','Euler 3','weights'});

**Is the crystal and specimen symmetry defined as hexagonal or orthorhombic?**

There are two states of the material, one is the annealed cold rolled sheet (from the results of EBSD, the texture is mainly the basal texture), the other is the annealed cold rolled sheet subjected to uniaxial tensile deformation.

best,

**Reply by ralfHielscher** [reply]
```matlab
ss = specimensSymmetry('orthorhombic')
cs = crystalSymmetry('hexagonal');
odf = ODF.load (fname,cs,ss,'resolution',res,'Bunge',...
'ColumnNames',{'Euler 1','Euler 2','Euler 3','weights'});
```
